ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -374,9 +366,11 @@ inline void OnScreenKeyboard::drawKBKeyFont(SkPoint index,bool onHLTile) {
                      draw co-ordiantes for any other return keys (other than deafult) to be done
                      here while drawing or can calculate during LAyout creation and maintained seperately
             */
+            if(onHLTile)  font.setSize(textHLFont_.getSize() * uniCharConfig.fontScaleFactor);
```

Comment:
as per coding guidelines, if and else blocks need to be under braces and in new line